frontend/apps/reader/modules/readerrolling.lua

```diff
@@ -1167,8 +1167,8 @@ function ReaderRolling:handleEngineCallback(ev, ...)
     -- ignore other events
 end
 
-local ENGINE_PROGRESS_INITIAL_DELAY = TimeVal:new{ sec = 2, usec = 0 }
-local ENGINE_PROGRESS_UPDATE_DELAY = TimeVal:new{ sec = 0, usec = 500000 }
+local ENGINE_PROGRESS_INITIAL_DELAY = time.s(2)
+local ENGINE_PROGRESS_UPDATE_DELAY = time.us(500000)
```

Comment:
Unless you explicitely want to keep the previous values as-is, `time.s(0.5)` reads easier than `time.us(500000)`

Reply:
Attention here: We can use 2.5 or 2.25 or 2.125, but *not* 2.1 (as this can not be encoded exactly to a float). So I find it better to use the `time.us` everywhere for fractional seconds.

Reply:
But it will just be a microsecond round error, right ? So, probably ignorable.
time.s(0.5) may give what time.us(500002), but we wouldn't really mind ? Or is it more critical that this rounding error?

Reply:
Yepp. But as I am changing many things here, I didn't want to introduce more complications as necessary. 
(and 2.5 would be exact.)

A shorter (and more readable) code can be in another follow-up cleanup PR.

Reply:
My vote would be for `time.ms(500)`, FWIW ;p.